server/models/Contest.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const contestSchema = new Schema({
+    title: {
+        type: String,
+        required: true,
+    },
+    description: {
+        type: String,
+        required: true
+    },
+    prize: {
+        type: Number,
+        required: true,
+    },
+    deadline: {
+        type: Date,
+        required: true
+    },
+    dateCreated: {
+        type: Date,
+        default: Date.now,
+    },
+})
```

Comment:
A contest might need a status attribute to show if it's in progress, completed, etc. Trying to do it by deadline would assume that they are automatically judged and winner chosen at the deadline